pkgs/applications/window-managers/sway/wrapper.nix

```diff
@@ -2,7 +2,7 @@
 , sway-unwrapped, swaybg
 , makeWrapper, symlinkJoin, writeShellScriptBin
 , withBaseWrapper ? true, extraSessionCommands ? "", dbus
-, withGtkWrapper ? false, wrapGAppsHook, gdk-pixbuf
```

Comment:
is sway being built without `gdk-pixbuf` support https://github.com/swaywm/sway/blob/876bc9e44c41661664aceee3cf4585b1586afbd9/meson_options.txt#L7?
You also need `librsvg` https://github.com/swaywm/sway/issues/3559#issuecomment-459989696, if you want svg support.

Reply:
Sway is built with `gdk-pixbuf`, I should add `librsvg` directly to the package not the wrapper, right ?

Reply:
Well, `gdk-pixbuf` actually propagates `librsvg`, so it should be no-op. But yes, you should add it to the unwrapped package. We need `gdk-pixbuf` here for its setup to be used with `wrapGAppsHook`.

Reply:
Okay, got it! Pushed modifications :)